
<!DOCTYPE html>
<html>

<head>

<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
<title>Matthew K Howard</title>
<link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>ðŸ§¬</text></svg>">

<style type="text/css">
body {
  font-family: Helvetica, arial, sans-serif;
  font-size: 14px;
  line-height: 1.6;
  padding-top: 10px;
  padding-bottom: 10px;
  background-color: white;
  padding: 30px; }

body > *:first-child {
  margin-top: 0 !important; }
body > *:last-child {
  margin-bottom: 0 !important; }

a {
  color: #4183C4; }
a.absent {
  color: #cc0000; }
a.anchor {
  display: block;
  padding-left: 30px;
  margin-left: -30px;
  cursor: pointer;
  position: absolute;
  top: 0;
  left: 0;
  bottom: 0; }

h1, h2, h3, h4, h5, h6 {
  margin: 20px 0 10px;
  padding: 0;
  font-weight: bold;
  -webkit-font-smoothing: antialiased;
  cursor: text;
  position: relative; }

h1:hover a.anchor, h2:hover a.anchor, h3:hover a.anchor, h4:hover a.anchor, h5:hover a.anchor, h6:hover a.anchor {
  background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA09pVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMy1jMDExIDY2LjE0NTY2MSwgMjAxMi8wMi8wNi0xNDo1NjoyNyAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvIiB4bWxuczp4bXBNTT0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wL21tLyIgeG1sbnM6c3RSZWY9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9zVHlwZS9SZXNvdXJjZVJlZiMiIHhtcDpDcmVhdG9yVG9vbD0iQWRvYmUgUGhvdG9zaG9wIENTNiAoMTMuMCAyMDEyMDMwNS5tLjQxNSAyMDEyLzAzLzA1OjIxOjAwOjAwKSAgKE1hY2ludG9zaCkiIHhtcE1NOkluc3RhbmNlSUQ9InhtcC5paWQ6OUM2NjlDQjI4ODBGMTFFMTg1ODlEODNERDJBRjUwQTQiIHhtcE1NOkRvY3VtZW50SUQ9InhtcC5kaWQ6OUM2NjlDQjM4ODBGMTFFMTg1ODlEODNERDJBRjUwQTQiPiA8eG1wTU06RGVyaXZlZEZyb20gc3RSZWY6aW5zdGFuY2VJRD0ieG1wLmlpZDo5QzY2OUNCMDg4MEYxMUUxODU4OUQ4M0REMkFGNTBBNCIgc3RSZWY6ZG9jdW1lbnRJRD0ieG1wLmRpZDo5QzY2OUNCMTg4MEYxMUUxODU4OUQ4M0REMkFGNTBBNCIvPiA8L3JkZjpEZXNjcmlwdGlvbj4gPC9yZGY6UkRGPiA8L3g6eG1wbWV0YT4gPD94cGFja2V0IGVuZD0iciI/PsQhXeAAAABfSURBVHjaYvz//z8DJYCRUgMYQAbAMBQIAvEqkBQWXI6sHqwHiwG70TTBxGaiWwjCTGgOUgJiF1J8wMRAIUA34B4Q76HUBelAfJYSA0CuMIEaRP8wGIkGMA54bgQIMACAmkXJi0hKJQAAAABJRU5ErkJggg==) no-repeat 10px center;
  text-decoration: none; }

h1 tt, h1 code {
  font-size: inherit; }

h2 tt, h2 code {
  font-size: inherit; }

h3 tt, h3 code {
  font-size: inherit; }

h4 tt, h4 code {
  font-size: inherit; }

h5 tt, h5 code {
  font-size: inherit; }

h6 tt, h6 code {
  font-size: inherit; }

h1 {
  font-size: 28px;
  color: black; }

h2 {
  font-size: 24px;
  border-bottom: 1px solid #cccccc;
  color: black; }

h3 {
  font-size: 18px; }

h4 {
  font-size: 16px; }

h5 {
  font-size: 14px; }

h6 {
  color: #777777;
  font-size: 14px; }

p, blockquote, ul, ol, dl, li, table, pre {
  margin: 15px 0; }

hr {
  background: transparent url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAYAAAAECAYAAACtBE5DAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAyJpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvIiB4bWxuczp4bXBNTT0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wL21tLyIgeG1sbnM6c3RSZWY9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9zVHlwZS9SZXNvdXJjZVJlZiMiIHhtcDpDcmVhdG9yVG9vbD0iQWRvYmUgUGhvdG9zaG9wIENTNSBNYWNpbnRvc2giIHhtcE1NOkluc3RhbmNlSUQ9InhtcC5paWQ6OENDRjNBN0E2NTZBMTFFMEI3QjRBODM4NzJDMjlGNDgiIHhtcE1NOkRvY3VtZW50SUQ9InhtcC5kaWQ6OENDRjNBN0I2NTZBMTFFMEI3QjRBODM4NzJDMjlGNDgiPiA8eG1wTU06RGVyaXZlZEZyb20gc3RSZWY6aW5zdGFuY2VJRD0ieG1wLmlpZDo4Q0NGM0E3ODY1NkExMUUwQjdCNEE4Mzg3MkMyOUY0OCIgc3RSZWY6ZG9jdW1lbnRJRD0ieG1wLmRpZDo4Q0NGM0E3OTY1NkExMUUwQjdCNEE4Mzg3MkMyOUY0OCIvPiA8L3JkZjpEZXNjcmlwdGlvbj4gPC9yZGY6UkRGPiA8L3g6eG1wbWV0YT4gPD94cGFja2V0IGVuZD0iciI/PqqezsUAAAAfSURBVHjaYmRABcYwBiM2QSA4y4hNEKYDQxAEAAIMAHNGAzhkPOlYAAAAAElFTkSuQmCC) repeat-x 0 0;
  border: 0 none;
  color: #cccccc;
  height: 4px;
  padding: 0;
}

body > h2:first-child {
  margin-top: 0;
  padding-top: 0; }
body > h1:first-child {
  margin-top: 0;
  padding-top: 0; }
  body > h1:first-child + h2 {
    margin-top: 0;
    padding-top: 0; }
body > h3:first-child, body > h4:first-child, body > h5:first-child, body > h6:first-child {
  margin-top: 0;
  padding-top: 0; }

a:first-child h1, a:first-child h2, a:first-child h3, a:first-child h4, a:first-child h5, a:first-child h6 {
  margin-top: 0;
  padding-top: 0; }

h1 p, h2 p, h3 p, h4 p, h5 p, h6 p {
  margin-top: 0; }

li p.first {
  display: inline-block; }
li {
  margin: 0; }
ul, ol {
  padding-left: 30px; }

ul :first-child, ol :first-child {
  margin-top: 0; }

dl {
  padding: 0; }
  dl dt {
    font-size: 14px;
    font-weight: bold;
    font-style: italic;
    padding: 0;
    margin: 15px 0 5px; }
    dl dt:first-child {
      padding: 0; }
    dl dt > :first-child {
      margin-top: 0; }
    dl dt > :last-child {
      margin-bottom: 0; }
  dl dd {
    margin: 0 0 15px;
    padding: 0 15px; }
    dl dd > :first-child {
      margin-top: 0; }
    dl dd > :last-child {
      margin-bottom: 0; }

blockquote {
  border-left: 4px solid #dddddd;
  padding: 0 15px;
  color: #777777; }
  blockquote > :first-child {
    margin-top: 0; }
  blockquote > :last-child {
    margin-bottom: 0; }

table {
  padding: 0;border-collapse: collapse; }
  table tr {
    border-top: 1px solid #cccccc;
    background-color: white;
    margin: 0;
    padding: 0; }
    table tr:nth-child(2n) {
      background-color: #f8f8f8; }
    table tr th {
      font-weight: bold;
      border: 1px solid #cccccc;
      margin: 0;
      padding: 6px 13px; }
    table tr td {
      border: 1px solid #cccccc;
      margin: 0;
      padding: 6px 13px; }
    table tr th :first-child, table tr td :first-child {
      margin-top: 0; }
    table tr th :last-child, table tr td :last-child {
      margin-bottom: 0; }

img {
  max-width: 100%; }

span.frame {
  display: block;
  overflow: hidden; }
  span.frame > span {
    border: 1px solid #dddddd;
    display: block;
    float: left;
    overflow: hidden;
    margin: 13px 0 0;
    padding: 7px;
    width: auto; }
  span.frame span img {
    display: block;
    float: left; }
  span.frame span span {
    clear: both;
    color: #333333;
    display: block;
    padding: 5px 0 0; }
span.align-center {
  display: block;
  overflow: hidden;
  clear: both; }
  span.align-center > span {
    display: block;
    overflow: hidden;
    margin: 13px auto 0;
    text-align: center; }
  span.align-center span img {
    margin: 0 auto;
    text-align: center; }
span.align-right {
  display: block;
  overflow: hidden;
  clear: both; }
  span.align-right > span {
    display: block;
    overflow: hidden;
    margin: 13px 0 0;
    text-align: right; }
  span.align-right span img {
    margin: 0;
    text-align: right; }
span.float-left {
  display: block;
  margin-right: 13px;
  overflow: hidden;
  float: left; }
  span.float-left span {
    margin: 13px 0 0; }
span.float-right {
  display: block;
  margin-left: 13px;
  overflow: hidden;
  float: right; }
  span.float-right > span {
    display: block;
    overflow: hidden;
    margin: 13px auto 0;
    text-align: right; }

code, tt {
  margin: 0 2px;
  padding: 0 5px;
  white-space: nowrap;
  border: 1px solid #eaeaea;
  background-color: #f8f8f8;
  border-radius: 3px; }

pre code {
  margin: 0;
  padding: 0;
  white-space: pre;
  border: none;
  background: transparent; }

.highlight pre {
  background-color: #f8f8f8;
  border: 1px solid #cccccc;
  font-size: 13px;
  line-height: 19px;
  overflow: auto;
  padding: 6px 10px;
  border-radius: 3px; }

pre {
  background-color: #f8f8f8;
  border: 1px solid #cccccc;
  font-size: 13px;
  line-height: 19px;
  overflow: auto;
  padding: 6px 10px;
  border-radius: 3px; }
  pre code, pre tt {
    background-color: transparent;
    border: none; }

sup {
    font-size: 0.83em;
    vertical-align: super;
    line-height: 0;
}

kbd {
  display: inline-block;
  padding: 3px 5px;
  font-size: 11px;
  line-height: 10px;
  color: #555;
  vertical-align: middle;
  background-color: #fcfcfc;
  border: solid 1px #ccc;
  border-bottom-color: #bbb;
  border-radius: 3px;
  box-shadow: inset 0 -1px 0 #bbb
}

* {
	-webkit-print-color-adjust: exact;
}
@media screen and (min-width: 914px) {
    body {
        width: 854px;
        margin:0 auto;
    }
}
@media print {
	table, pre {
		page-break-inside: avoid;
	}
	pre {
		word-wrap: break-word;
	}
}
</style>


</head>

<body>

<h1 id="toc_0">Matthew K. Howard</h1>

<h5 id="toc_1"><strong>Last updated: November 2025</strong></h5>

<h2 id="toc_3">about</h2>

<p>Hello! My name is Matt, and this is my sandbox on the internet. I am currently a PhD candidate at UCSF working in the <a href="https://www.wcoyotelab.com"target="_blank" rel="noopener noreferrer">Coyote-Maestas</a> and <a href="https://mangliklab.com"target="_blank" rel="noopener noreferrer">Manglik</a> labs. Prior to UCSF, I worked in the <a href="https://sites.wustl.edu/jackrel/"target="_blank" rel="noopener noreferrer">Jackrel</a> lab at WashU.</p>

<p>My current work uses deep mutational scanning and other library-based approaches to understand GPCR function, pharmacology, and evolution.</p>

<p>If you're interested, I can be found the following places: <a href="mailto:matthewkarlhoward@gmail.com?subject=Hello"target="_blank" rel="noopener noreferrer">email me</a>, <a href="https://x.com/matthewkhoward"target="_blank" rel="noopener noreferrer">Twitter(X)</a>, <a href="https://bsky.app/profile/matthewkhoward.bsky.social"target="_blank" rel="noopener noreferrer">Bluesky</a>, <a href="https://github.com/matthewkarlhoward"target="_blank" rel="noopener noreferrer">Github</a>, <a href="https://www.linkedin.com/in/matthewkarlhoward/"target="_blank" rel="noopener noreferrer">Linkedin</a>, <a href="https://scholar.google.com/citations?user=bOu5znYAAAAJ&hl=en"target="_blank" rel="noopener noreferrer">Google Scholar</a>, <a href="https://orcid.org/0000-0002-2128-2065"target="_blank" rel="noopener noreferrer">Orcid</a>.</p>

<h2 id="toc_5">publications</h2>

<p>Please find a list of my publications below. I try to keep this up to date, but also check out my <a href="https://scholar.google.com/citations?user=bOu5znYAAAAJ&hl=en" target="_blank" rel="noopener noreferrer">Google Scholar page</a>. Please <a href="mailto:matthewkarlhoward@gmail.com?subject=Hello">email me</a> if you run into paywall issues.</p>

<h3>PhD work at UCSF</h3>
<ul>
  <li>[in prep] <strong>Howard MK</strong> et al (2026) Deep mutational pharmacology reveals the molecular basis of opioid agonism.</li>

  <li>[Submitted - in review] <strong>Howard MK</strong> & Coyote-Maestas W (2026) Mechanistic mutational scanning to uncover the secret life of proteins. <em>Annual Reviews Biomedical Data Science</em>.</li>

  <li>Tedman A, Goel M, Shah S, <strong>Howard MK</strong>, Chamness LM, Bonifasi A, Adams I, Gallagher JM, Penn WD, Nemec K, McDonald EF, Corman BN, Robinson JP, Post CB, Clark PL, Babu MM, Manglik AM, Kuntz CP, Coyote-Maestas W, Schlebach JP (2025) Deep Receptor Scanning Reveals General Sequence Constraints on GPCR Biosynthesis. <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2025.09.19.677468" target="_blank" rel="noopener noreferrer">https://doi.org/10.1101/2025.09.19.677468</a></li>

  <li>Freudenberg J, Rao J, <strong>Howard MK</strong>, Macdonald C, Greenwald NF, et al. (2025) Accurate variant effect estimation in FACS-based deep mutational scanning data with Lilace. <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2025.06.24.661380" target="_blank" rel="noopener noreferrer">https://doi.org/10.1101/2025.06.24.661380</a></li>

  <li>Rao J, Wang M, <strong>Howard MK</strong>, Coyote-Maestas W, Pimentel H (2025) Cosmos: A Position-Resolution Causal Model for Direct and Indirect Effects in Protein Functions. <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2025.08.01.667517" target="_blank" rel="noopener noreferrer">https://doi.org/10.1101/2025.08.01.667517</a></li>

  <li>Rao J, Wang M, <strong>Howard MK</strong>, Macdonald C, Fraser JS, Coyote-Maestas W, Pimentel H (2025) Rosace-AA: Enhancing interpretation of deep mutational scanning data with amino acid substitution and position-specific insights. <em>Bioinformatics Advances</em>. <a href="https://doi.org/10.1093/bioadv/vbaf218" target="_blank" rel="noopener noreferrer">https://doi.org/10.1093/bioadv/vbaf218</a></li>

  <li>Macdonald C, Nedrud D, Grimes PR, Trinidad D, Shin C, Greenwald N, <strong>Howard MK</strong>, Fraser J, Coyote-Maestas (2025) DIMPLE library generation and assembly protocol V6. <em>protocol.io</em>. <a href="https://dx.doi.org/10.17504/protocols.io.rm7vzy7k8lx1/v6" target="_blank" rel="noopener noreferrer">https://doi.org/10.1101/2024.10.10.617568</a></li>

  <li>Naghipourfar M, Chen S, <strong>Howard MK</strong>, Macdonald CB, Saberi A, Hagen T, Mofrad MRK, Coyote-Maestas W, Growald M (2024) A Suite of Foundation Models Captures the Contextual Interplay Between Codons. <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2024.10.10.617568" target="_blank" rel="noopener noreferrer">https://doi.org/10.1101/2024.10.10.617568</a></li>

  <li><strong>Howard MK</strong>, Hoppe N, Huang XP, Mitrovic D, BillesbÃ¸lle CB, Macdonald CB, Mehrotra E, Rockefeller Grimes P, Trinidad DD, Delemotte L, English J, Coyote-Maestas W, Manglik A (2024) Molecular basis of proton-sensing by G protein-coupled receptors. <em>Cell</em>. <a href="https://doi.org/10.1016/j.cell.2024.11.036" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.cell.2024.11.036</a></li>

  <li>Rao J, Xin R, Macdonald C, <strong>Howard MK</strong>, Estevam GO, Yee SW, Wang M, Fraser JS, Coyote-Maestas W, Pimentel H (2024) Rosace: a robust deep mutational scanning analysis framework employing position and mean-variance shrinkage. <em>Genome Biology</em>. <a href="https://doi.org/10.1186/s13059-024-03279-7" target="_blank" rel="noopener noreferrer">https://doi.org/10.1186/s13059-024-03279-7</a></li>

  <li>Polacco BJ, Lobingier BT, Blythe EE, Abreu N, Khare P, <strong>Howard MK</strong>, Gonzalez-Hernandez AJ, Xu J, Li Q, Novy B, Naing ZZC, Shoichet BK, Coyote-Maestas W, Levitz J, Krogan NJ, Zastrow MV, HÃ¼ttenhain R (2024) Profiling the proximal proteome of the activated Î¼-opioid receptor. <em>Nature Chemical Biology</em>. <a href="https://doi.org/10.1038/s41589-024-01588-3" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41589-024-01588-3</a></li>
</ul>

<h3>From WUSTL</h3>
<ul>
  <li>Chen S, Puri A, Bell B, Fritsche J, Palacios HH, Balch M, Sprunger ML, <strong>Howard MK</strong>, Patterson JN, Patti GJ, Davis AA, Jackrel ME (2024) HtrA1 disaggregates Î±-synuclein amyloid fibrils and converts them into non-toxic and seeding incompetent species. <em>Nature Communications</em>. <a href="https://doi.org/10.1038/s41467-024-46538-8" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41467-024-46538-8</a></li>

  <li>Shrimali PC, Chen S, Das A, Dreher R, <strong>Howard MK</strong>, Ryan J, Buck J, Kim D, Sprunger M, Rudra JS, Jackrel ME (2023) Amyloidogenic Propensity of Self-Assembling Peptides and their Adjuvant Potential for use as DNA Vaccines. <em>Acta Biomaterialia</em>. <a href="https://doi.org/10.1016/j.actbio.2023.08.015" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.actbio.2023.08.015</a></li>

  <li><strong>Howard MK</strong>, Miller KR, Sohn BS, Ryan JJ, Xu A, Jackrel ME (2023) Probing the drivers of <em>Staphylococcus aureus</em> biofilm protein amyloidogenesis and disrupting biofilms with engineered protein disaggregases. <em>mBio</em>. <a href="https://doi.org/10.1128/mbio.00587-23" target="_blank" rel="noopener noreferrer">https://doi.org/10.1128/mbio.00587-23</a></li>

  <li><strong>Howard MK</strong>, Sohn BS, von Borcke J, Xu A, Jackrel ME (2020) Functional analysis of proposed substrate-binding residues of Hsp104. <em>PLoS ONE</em>. <a href="https://doi.org/10.1371/journal.pone.0230198" target="_blank" rel="noopener noreferrer">https://doi.org/10.1371/journal.pone.0230198</a></li>
</ul>

<h2 id="toc_4">news</h2>

<ul>
  <li>May 2024: Jingyou's paper is out in Genome Biology</li>
  <li>April 2024: Matt's paper is out on Biorxiv</li>
  <li>March 2024: Matt gives an invited talk at the 2024 Ligand Gating and Molecular Recognition GRS and a poster at the GRC (Ventura, CA)</li>
  <li>March 2024: Andy's paper is out in Nature Communications</li>
  <li>December 2023: Ruth's Î¼OR-APEX paper is out on Biorxiv</li>
  <li>October 2023: Jingyou's ROSACE paper is out on Biorxiv</li>
  <li>June 2023: Matt presents a poster at the 2023 Membrane Protein Folding GRC</li>
  <li>June 2023: Matt is awarded the UCSF Julius R. and Patricia A. Krevans Fellowship</li>
  <li>April 2023: Matt presents ongoing work at the UCSF GPCR Supergroup monthly meeting (San Francisco, CA)</li>
  <li>April 2023: Matt's paper from the Jackrel Lab studying the PSMÎ± peptides is accepted at mBio!</li>
  <li>April 2023: Matt gives an invited talk at the UCSF Cell Engineering & Synthetic Biology seminar series (San Francisco, CA)</li>
  <li>January 2023: Matt gives an invited short talk and poster presentation at the 2023 GPCR Workshop (Kona, HI) and is supported by the UCSF Graduate Division Travel Award</li>
  <li>October 2022: Matt gives a talk and poster presentation at the CZ Biohub Inter-lab Confab (San Francisco, CA)</li>
  <li>October 2022: Matt give a poster presentation on his work at the annual UCSF Tetrad retreat (Tahoe, CA)</li>
  <li>June 2022: Matt Joins the Willow Coyote-Maestas and Manglik Labs</li>
  <li>April 2022: 3rd rotation with the Mordes Lab</li>
  <li>January 2022: 2nd rotation with Willow Coyote-Maestas and the Manglik Lab</li>
  <li>September 2021: 1st lab rotation in the Fraser Lab</li>
  <li>September 2021: First day at UCSF!</li>
  <li>April 2021: Matt accepts an offer to join the Tetrad Program at UCSF!</li>
  <li>January 2021: Matt serves as a TA for Introduction to Biochemistry (Bio406)</li>
  <li>March 2020: COVID-19 starts to turn the world inside-out</li>
  <li>March 2020: Matt's work on Hsp104 pore loops is published in PLOS One</li>
  <li>November 2019: Matt presents a poster at the Center for Science & Engineering of Living Systems (CSELS) meeting</li>
  <li>May 2019: Matt graduates from Washington University in St. Louis with majors in both Biology and Chemistry: concentration in Biochemistry</li>
  <li>May 2019: Matt receives the Lipkin Award for outstanding undergraduate research in chemistry</li>
  <li>May 2019: Matt receives the Outstanding PLTL Award for excellence in academic mentorship</li>
  <li>April 2019: Matt presents a poster on his work from the Jackrel Lab and another poster from his work in Bio4522 at the Undergraduate Research Symposium</li>
  <li>November 2018: Matt presents his work from the Jackrel Lab at the Undergraduate Research Symposium</li>
  <li>August 2018: Matt serves as a TA for General Biochemistry I (Chem481/Bio4810)</li>
  <li>July 2018: Matt attends the Alpha Chi Sigma Chemistry Fraternity Conclave in Pittsburgh, PA</li>
  <li>April 2018: Matt received a BioSURF award to fund his summer research in the Jackrel Lab</li>
  <li>August 2017: Matt starts work as an undergraduate research assistant in the Jackrel Lab at Washington University in St. Louis</li>
  <li>January 2017: Matt starts work as a undergraduate research assistant in the Wayne Barnes Lab at the Washington University School of Medicine</li>
  <li>August 2016: Matt starts work as a General Chemistry (Chem111/112) Peer-Led Team Learning (PLTL) Leader</li>
  <li>June 2016: Matt starts a summer research internship with the Carolinas Laparoscopic and Advanced Surgery Program (CLASP) at Carolinas Healthcare System</li>
  <li>August 2015: Matt starts his undergraduate degree at Washington University in St. Louis</li>
  <li>June 2015: Matt graduates from Providence Day School (Charlotte, NC)</li>
  <li>March 2013: Matt received an honorable mention by the Toshiba|NSTA Exploravision competition for his project "Utilizing an Automatically Titrating Therapeutic Oxygen Delivery Device (ATTODD) to Improve Individualized Patient Care"</li>
  <li>June 2012: Matt, Alex, and Christian win 1st at the state and regional level and travel to Washington, DC to compete for the national eCybermission title</li>
  <li>February 2012: Matt, Emily, and Alexander are invited to the White House Science Fair to present their eCybermission project on water filtration</li>
  <li>June 2011: Matt, Emily, and Alexander win 1st at the state and regional level and travel to Washington, DC to compete for the national eCybermission title</li>
</ul>

</body>

</html>
